Scripts/Debug/Yotam LCA Model.py

```diff
@@ -455,24 +450,24 @@ def evaluate_net_perf_lca(mnet_lca, test_tasks, all_tasks, num_features, num_inp
 
 	# Instantiate test matrices
 	input_test_pts, output_true_pts, control_test_pts = generate_testing_data(test_tasks,
-																   all_tasks, 
-																   num_features, 
-																   num_input_dims, 
-																   num_output_dims, 
+																   all_tasks,
+																   num_features,
+																   num_input_dims,
+																   num_output_dims,
 																   num_test_points=num_test_points)
 
 	# Run the outer composition, one point at a time (for debugging purposes)
 	for i in range(num_test_points):
 		# Construct input dict
-		input_set = { 
+		input_set = {
 			'inputs' : {
 				mnet_lca.nodes['mnet'].nodes['input'] : input_test_pts[i, :].tolist(),
 				mnet_lca.nodes['mnet'].nodes['control'] : control_test_pts[i, :].tolist()
 			}
 		}
 
 		try:
-			mnet_lca.run( { mnet_lca.nodes['mnet'] : input_set } )
+			mnet_lca.run( { mnet_lca.nodes['mnet'] : input_set['inputs'] })
 		except Warning:
```

Comment:
This doesn't work with the filter at the top removed. Fixing the 'message' regexp should restore the functionality.